RIOT-OS/RIOT
# tests: use conversion macros and proper suffixes

### Contribution description

This PR changes the value suffix from `U` to `UL` for time conversion macros.
Further it adapts timer values in tests to either use these macros or have proper suffixes to get correct values across all platforms. Specifically on 8 and 16 bit platforms such as Atmega based Arduinos.

### Testing procedure

Run various tests that use timer values, these should still work on any platform as before but should now also work correctly on lower bit platforms.


### Issues/PRs references

Fixes #7301
